pom.xml

```diff
@@ -50,6 +63,17 @@
       <groupId>org.folio</groupId>
       <artifactId>folio-spring-base</artifactId>
       <version>${folio-spring-base.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.springframework.boot</groupId>
+          <artifactId>spring-boot-starter-data-jpa</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
```

Comment:
hello @azizbekxm do you really need this dependency?

Reply:
these dependencies added from Thunderjet developer and it solved above test error, so it needs